pipelines/batch/src/main/java/org/openmrs/analytics/JdbcFetchHapi.java

```diff
@@ -83,54 +110,148 @@ public HapiRowDescriptor mapRow(ResultSet resultSet) throws Exception {
 
     private final String query;
 
+    private final String tagQuery;
+
     public FetchRowsJdbcIo(JdbcIO.DataSourceConfiguration dataSourceConfig, String since) {
       this.dataSourceConfig = dataSourceConfig;
       // Note the constraint on `res.res_ver` ensures we only pick the latest version.
       StringBuilder builder =
           new StringBuilder(
```

Comment:
I don't know Java, but I wonder if we can use TextBlocks with """

Reply:
I am not sure I followed, do you mean instantiating with ""? like 
`private final String tagQuery = ""` If yes, we do that when we haven't made the variable final. With final variables, we have to set its value in the constructor otherwise compiler complains

Reply:
Ah sorry.  Let me clarify. I was just wondering if Java supports multiline strings natively.  It looks like there's something called TextBlocks, but that's only supported after JDK13, which I'm not sure that we use.

https://openjdk.org/jeps/378

e.g.

```
String s = """
           text
           text
           text
           """;
```

Reply:
resolving this since its a no-op